content/md/imports-exports/quick-export.md

```diff
@@ -43,6 +43,10 @@ Only the working channel is taken into account, and thus all the locale informat
 
 For this export from the grid, the completeness, the product status, their classification are not taken into account to determine what information can be exported. All the products and product models selected in the product grid, regardless of their details, are exported to the CSV file or XLSX file.
 
+::: warning
+Note that the following columns are not exported during a Grid Context Export: Labels, Image, Completeness, "Created at", "Updated at" and Variant products.
```

Comment:
you can remove "during a Grid Context Export" because it's not exported also if you choose all attributes